project-generator/src/main/resources/io/syndesis/project/converter/templates/pom.xml.mustache

```diff
@@ -16,6 +16,7 @@
     <camel.version>@camel.version@</camel.version>
     <funktion.version>@funktion.version@</funktion.version>
     <atlasmap.version>@atlasmap.version@</atlasmap.version>
+    <syndesis-connectors.version>@syndesis-connectors.version@</syndesis-connectors.version>
```

Comment:
It's not AFAI can tell, but these is this test:

https://github.com/KurtStam/syndesis-rest/blob/master/project-generator/src/test/java/io/syndesis/project/converter/DefaultProjectGeneratorTest.java#L150

Not sure why that test is there though, but I thought to play is safe.